src/Normalize.ml

```diff
@@ -159,3 +159,43 @@ let equal_impl ast1 ast2 = Poly.equal (impl ast1) (impl ast2)
 let equal_intf ast1 ast2 = Poly.equal (intf ast1) (intf ast2)
 
 let equal_use_file ast1 ast2 = Poly.equal (use_file ast1) (use_file ast2)
+
+exception Formatting_disabled
+
+let iter =
+  let structure_item _ (si: structure_item) =
+    match si.pstr_desc with
+    | Pstr_attribute ({txt= "ocamlformat.disable"; _}, _) ->
+        raise Formatting_disabled
+    | _ -> ()
+  in
+  let signature_item _ (si: signature_item) =
+    match si.psig_desc with
+    | Psig_attribute ({txt= "ocamlformat.disable"; _}, _) ->
+        raise Formatting_disabled
+    | _ -> ()
+  in
+  {Ast_iterator.default_iterator with structure_item; signature_item}
```

Comment:
It seems that `Ast_iterator` is not handled by ocaml-migrate-parsetree, so this change causes compilation failure for ocaml <> 4.07. Perhaps it can be added to OMP: https://github.com/ocaml-ppx/ocaml-migrate-parsetree/issues/47 .

Reply:
https://github.com/ocaml-ppx/ocamlformat/pull/252